lib/matplotlib/backends/backend_qt5cairo.py

```diff
@@ -19,8 +19,8 @@ def draw(self):
     def paintEvent(self, event):
         self._update_dpi()
         dpi_ratio = self._dpi_ratio
-        width = dpi_ratio * self.width()
-        height = dpi_ratio * self.height()
+        width = int(dpi_ratio * self.width())
```

Comment:
We may want to use `round` here rather than `int` to move the jumps away from the integers.

However, this may have issues with not being in sync with what we do else were in the code (which is `int`).